narwhals/_dask/group_by.py

```diff
@@ -36,14 +36,38 @@ def agg(s0: pd.core.groupby.generic.SeriesGroupBy) -> int:
     )
 
 
+def var(
+    ddof: int = 1,
+) -> Callable[
+    [pd.core.groupby.generic.SeriesGroupBy], pd.core.groupby.generic.SeriesGroupBy
+]:
+    from functools import partial
+
+    import dask_expr as dx
+
+    return partial(dx._groupby.GroupBy.var, ddof=ddof)
```

Comment:
The minimum version we support for dask is `"dask[dataframe]==2024.7"`, which installs `dask-expr==1.1.8`, and `dx._groupby.GroupBy` is already available.

Reply:
@phofl is there a public variant of this?